nest-forms-backend/Dockerfile

```diff
@@ -1,65 +1,54 @@
-ARG FORMS_SHARED_TAG=latest
+# id forms-shared tag is not specified, use general "latest"
+ARG FORMS_SHARED_TAG=6869d7843b1710ea921281de07182ddbd977c37e
```

Comment:
latest?

Reply:
Sorry forgot to remove the tag which i was testing it